docs/src/modules/javascript/pages/proto.adoc

```diff
@@ -5,6 +5,38 @@ include::partial$attributes.adoc[]
 
 Akka Serverless SDKs support `protobuf` descriptors in the https://developers.google.com/protocol-buffers/docs/proto3[Proto3 {tab-icon}, window="new-tab"] Protocol Buffers language. You define command messages, data associated with Entities, and events in `.proto` files. From these definitions, the gRPC compiler creates client and server side code that saves work for you and enables Akka Serverless to serialize message data.
 
+Note that Value Entities and Event Sourced Entities are slightly different based the codegen requirements. Other than the code generation annotations, the proto file are materially equivalent for the same use-case. See the following examples which show the difference.
```

Comment:
Not sure what "codegen requirements" are? or "code generation annotations"? And, can we pull the code in from the examples directory instead of inline?

Reply:
Ok, I will tweak that text. I'll have to alter the proto files. The VE is tagged but the section is too large so I'll add tags for a tighter extraction. The ESE is not tagged at all.  I believe tags can be nested within other tags, Correct? We'll see.

Reply:
@pvlugter Hi Peter - I'm trying to pull in:  include::example$js-eventsourced-shopping-cart/proto/shoppingcart_domain.proto[tag=proto_state]
But that sample does not appear to be getting dynamically copied into examples. Could you check this out and fix if possible - or direct me to what I am missing. Thanks.

Reply:
@thothmoses, in order to test this, you'll need to follow the procedure I sent in email the other day. The samples are built in the docs/current branch.

Reply:
@rstento  Thanks, but Nope, that is not the problem all the other samples are being pulled in and are appearing after running Make.
It is just this one file is throwing an error during the build. @pvlugter @patriknw Can you figure this?

Reply:
@thothmoses, right, that sample is not currently being copied. Only these samples:

https://github.com/lightbend/akkaserverless-javascript-sdk/blob/f7b529a2820700130e0c25bd036345a692b5b6dd/docs/Makefile#L43-L45

You can add the appropriate line to the Makefile:

```
rsync -a --exclude-from=.examplesignore ../samples/js-eventsourced-shopping-cart/ "${managed_examples}/js-eventsourced-shopping-cart/"
```

And check what ends up in `docs/build/src/managed/modules/javascript/examples/` after `make examples`

Reply:
@pvlugter Perfect, thank you sir. That worked, and I learned something very useful.